src/Services/Settings.js

```diff
@@ -11,13 +11,18 @@ const initialSettings = {
   verboseLogging: false,
   firstRun: true,
   randomizeMnemonicOnStart: false,
+  logsDirectory: null,
   server: {
     hostname: "127.0.0.1",
     port: 7545,
     network_id: 5777,
+    default_balance_ether: 100,
     total_accounts: 10,
     unlocked_accounts: [],
-    vmErrorsOnRPCResponse: true
+    locked: false,
```

Comment:
FWIW, it should be possible to read the `server` object back out of `ganache-core` now with the default options applied. I made that happen so that we didn't have to worry about keeping this in sync w/ the other module.

Reply:
Let's do that on a separate task rather than earmark it here

Reply:
I go back and forth about this sort of thing... I'm fine with it being a separate task if it actually gets done, but there's really never any impetus to prioritize such changes as separate standalone tasks until they become so numerous that big refactorings are warranted. Judicious inclusion of related-but-out-of-scope improvements can be a good way to keep those sort of things at bay.

I'm happy either way, though - if you want to make the improvement, great. If not, I'll probably scratch that itch at some point or another.

Reply:
I'll add it to the list of things for me to do and it'll be addressed in priority order! i'll try to not let this one fall down the cracks. I was a little unhappy with how defaults were applied, so unless you fixed that, I'll definitely bring this up again haha